decidim-meetings/app/controllers/decidim/meetings/meetings_controller.rb

```diff
@@ -18,15 +18,15 @@ def static_map
       private
 
       def meetings
-        @meetings ||= search.results.page(params[:page]).per(12)
+        @meetings ||= search.results.order("start_time DESC").page(params[:page]).per(12)
```

Comment:
When upcoming, order should be `asc`. When `past`, order should be `desc` (most recent first).